tests/apps/lwaftr/data/Makefile

```diff
@@ -0,0 +1,18 @@
+#
+# Makefile
+# Adrian Perez, 2015-10-01 14:56
+#
```

Comment:
Other Makefile files like `/Makefile` or `src/Makefile` don't have this type of information on the top. Although this is a Makefile for our own use, I'd respect this convention.